js/ui/scroll_view/ui.scrollable.simulated.js

```diff
@@ -821,7 +818,21 @@ var SimulatedStrategy = Class.inherit({
     _createActionHandler: function(optionName) {
         var that = this,
             actionHandler = that._createActionByOption(optionName);
-        return function() {
+
+        if(optionName === "onScroll") {
+            actionHandler = that._createActionByOption("onScroll", {
+                afterExecute: function(e) {
```

Comment:
```suggestion
                afterExecute: (e) => {
```
Then you can use `this` instead of `that`